econml/sklearn_extensions/model_selection.py

```diff
@@ -367,17 +378,9 @@ def _cross_val_predict(estimator, X, y=None, *, groups=None, cv=None,
     # independent, and that it is pickle-able.
     parallel = Parallel(n_jobs=n_jobs, verbose=verbose,
                         pre_dispatch=pre_dispatch)
-    prediction_blocks = parallel(delayed(_fit_and_predict)(
+    predictions = parallel(delayed(_fit_and_predict)(
```

Comment:
Please add a TODO or other comment mentioning this (so that we can find it easily if we want to try to conditionally address it), but then that's fine.

Reply:
Done